rhel_osv/osv.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2024 Jason Shepherd
+# License: GPLv3+
+
+import datetime
+import re
+
+from rhel_osv.csaf import CSAF, Reference, Remediation
+
+class Event(object):
+    """
+    class to hold event information for a Range. Advisories for Red Hat RPM based products always
+    assume all previous versions are affected.
+    """
+    def __init__(self, fixed: str):
+        self.introduced = "0"
+        self.fixed = fixed
+
+class Range(object):
```

Comment:
https://github.com/ossf/osv-schema/issues/252 for awareness...

Reply:
Added a comment there. Staying with ECOSYSTEM is probably also fine.